go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/pingcap/errors v0.11.4
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.0
+	github.com/stretchr/testify v1.8.0
```

Comment:
`testify` is currently used only in `baggage_test.go`, and also used by a bunch of dependencies.